packages/exchange-ui-core/env.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+REQUIRED_VARIABLES=(
+  MODE
+  WEB3
+  BACKEND_URL
+  BACKEND_PORT
+  BLOCKCHAIN_EXPLORER_URL
+  REGISTRATION_MESSAGE_TO_SIGN
+  ISSUER_ID
+  DEVICE_PROPERTIES_ENABLED
+  DEFAULT_ENERGY_IN_BASE_UNIT
+  EXCHANGE_WALLET_PUB
+  GOOGLE_MAPS_API_KEY
+)
```

Comment:
Here you probably need only EXCHANGE_WALLET_PUB only

Reply:
Maybe DEFAULT_ENERGY_IN_BASE_UNIT also?